### DISCUSSION TITLE
Is it possible to intercept http request?

### DISCUSSION THREAD

**Discussion by ardzz** [php-webdriver/php-webdriver#939] [Q&A]
I wont intercept every request and modify automatically, is that possible?

**Comment by yunsiong** [comment]
Can intercept the request but can't modify the request because cannot get the requestId to send commad Fetch.continueRequest or Fetch.fulfillRequest. There is no way to add event listenser for Fetch.requestPaused and the requestId parameter require when sending modify the intercept request command. 
The code below is to intercept the request.
            $chromedevtools =  new ChromeDevToolsDriver($this->driver);
            $chromedevtools->execute('Fetch.enable',['patterns' => [(object)['urlPattern' =>'*v1/public/init*']]]);
